lms/djangoapps/teams/csv.py

```diff
@@ -19,3 +26,234 @@ def load_team_membership_csv(course, response):
         "Team membership CSV download is not yet implemented."
     )
     response.write(not_implemented_message + "\n")
+
+
+class TeamMembershipImportManager(object):
+    """
+    A manager class that is responsible the import process of csv file including validation and creation of
+    team_courseteam and teams_courseteammembership objects.
+    """
+
+    def __init__(self, course):
+        self.validation_errors = []
+        self.teamset_ids = []
+        self.user_ids_by_teamset_id = {}
+        self.teamset_ids = []
+        self.number_of_record_added = 0
+        self.course = course
+        self.max_errors = 0
```

Comment:
There's no way to modify this value for a given instance of the class - maybe we can do without it for now, and worry about allowing some configurable number of validation errors later?

Reply:
I don't mind removing it, but as of now, the code is written around having an error threshold. I would rather keep it set to some value other than 0 vs rewriting things.

Reply:
OK, I think it's fine to leave it as-is - we can re-assess later if requirements change.